scripts/compare-motion-clips.py

```
@@ -0,0 +1,116 @@
+        px = im.load()
+        out = []
+        for r in range(rows):
+            row = []
+            for c in range(cols):
+                pts = [(x - c * cw, y - r * ch)
+                       for y in range(r * ch, (r + 1) * ch)
+                       for x in range(c * cw, (c + 1) * cw) if px[x, y] > 45]
+                if not pts:
+                    row.append(frozenset())
+                    continue
+                xs = [p[0] for p in pts]; ys = [p[1] for p in pts]
```

Ruff (0.15.20)

[error] Multiple statements on one line (semicolon)

(E702)